### PR TITLE
Feature: Normalize docs

### PR DESCRIPTION
**Pull request by f3rno**
### Description:
Updates documentation as per the new spec + update with local changes (minimal)

### PR status:
- [x] Version bumped
- [x] Change-log updated

### CODE REVIEW

File: lib/data/symbols.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+// TODO: Update
```
Comment (JacobPlaster) [inline lib/data/symbols.js]: TODO? Do we need these symbols?

File: CHANGELOG [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# 1.0.4
+- docs: create/update
+
+# 1.0.3
+- add missing debug dep
+- minor refactoring
+
+# 1.0.2
+- added LICENSE
+- onExit: reduce grace period to 3d
```
Comment (JacobPlaster) [inline CHANGELOG]: `3d` -> `3s`

File: CHANGELOG [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+# 1.0.1
+- removed package-lock.json
+- removed babel dep
+
+# 1.0.0
+- initial release
```
Comment (JacobPlaster) [inline CHANGELOG]: (minor) new line at end of file

File: lib/data/symbols.js (hunk 1 above)
Comment (f3rno) [reply lib/data/symbols.js]: They are used to provide constants, i.e. `SYMBOLS.BTC_USD` here:
```js
const market = {
  symbol: SYMBOLS.BTC_USD,
  tf: TIME_FRAMES.ONE_HOUR
}
```

Do you think we should remove them? Or maybe optionally load the list when the file is required() and fallback to the hardcoded list if there is no network? Would be simple.

File: CHANGELOG (hunk 2 above)
Comment (f3rno) [reply CHANGELOG]: Fixed

File: CHANGELOG (hunk 3 above)
Comment (f3rno) [reply CHANGELOG]: Fixed

File: lib/data/symbols.js (hunk 1 above)
Comment (JacobPlaster) [reply lib/data/symbols.js]: RIght I see, we do something similar for golang. Nah lets keep it like this, im a bit worried the network loading version would be a bit too 'magical' and might cause for some nasty error debugging

**Pull request merged**